src/Components/Form/FormFields/AutoExpandingTextInputFormField.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect, useRef } from "react";
+import TextAreaFormField, { TextAreaFormFieldProps } from "./TextAreaFormField";
+
+type AutoExpandingTextInputFormFieldProps = TextAreaFormFieldProps & {
+  maxHeight?: number;
+};
+
+const AutoExpandingTextInputFormField = (
+  props: AutoExpandingTextInputFormFieldProps
+) => {
+  const myref = useRef<HTMLTextAreaElement>(null);
+  useEffect(() => {
+    if (myref.current == null) return;
+    const text = myref.current.textContent?.split("\n");
+    const len = (text?.length == 0 ? 1 : text?.length) || 1;
```

Comment:
`text?.length == 0` this will never be true, as even for an empty text input, when you split `textContent` it in previous line, there will be at least one element. Simplify this condition accordingly.

```
> ("".split("\n")).length
1
```